zaza/openstack/charm_tests/neutron/tests.py

```diff
@@ -318,6 +318,253 @@ def test_security_checklist(self):
                 expected_to_pass=expected_to_pass)
 
 
+class NeutronOpenvSwitchTest(test_utils.OpenStackBaseTest):
+    """Test basic Neutron Openvswitch Charm functionality."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Run class setup for running Neutron Openvswitch tests."""
+        super(NeutronOpenvSwitchTest, cls).setUpClass()
+
+        cls.current_os_release = openstack_utils.get_os_release()
+
+        cls.compute_unit = zaza.model.get_units('nova-compute')[0]
+        cls.rabbitmq_unit = zaza.model.get_units('rabbitmq-server')[0]
+        cls.neutron_api_unit = zaza.model.get_units('neutron-api')[0]
+        cls.n_ovs_unit = zaza.model.get_units('neutron-openvswitch')[0]
+
+        cls.bionic_stein = openstack_utils.get_os_release('bionic_stein')
+        cls.trusty_mitaka = openstack_utils.get_os_release('trusty_mitaka')
+
+        if cls.current_os_release >= cls.bionic_stein:
+            cls.pgrep_full = True
+        else:
+            cls.pgrep_full = False
+
+        # set up client
+        cls.neutron_client = (
+            openstack_utils.get_neutron_session_client(cls.keystone_session))
+
+    def test_100_services(self):
```

Comment:
Are these tests named like this `test_nnn...`  so that it attempts to order the tests somehow?

Reply:
they are run in the order that are listed in the code. It's the conversion from amulet, and that's how they were named there, so just following the same set of functions